dashboard/client/components/pod-ports.scss

```diff
@@ -19,6 +19,6 @@
   .Spinner {
     --spinner-size: #{$unit * 2};
     margin-left: $margin;
-    position: absolute;
+    position: static;
```

Comment:
Without this change the spinner does not appear for containers in pods port-forwarding (still works for service port-forwarding). There may be a better solution?